mc/Controllers/APIController.cs

```diff
@@ -101,14 +101,20 @@ public ActionResult Ranker(string defaultActions)
             {
                 APIUtil.Authenticate(this.Request);
 
-                int[] defaultActionArray = Array.ConvertAll(defaultActions.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries), Convert.ToInt32);
-
                 var client = DecisionServiceClientFactory.AddOrGetExisting(ModelSuccessNotifier);
                 var context = APIUtil.ReadBody(this.Request);
                 var eventId = APIUtil.CreateEventId();
-                var actions = defaultActionArray != null && defaultActionArray.Length > 0 ?
-                        client.ChooseRanking(eventId, context, defaultActionArray) :
-                        client.ChooseRanking(eventId, context);
+
+                int[] actions;
+                if (string.IsNullOrWhiteSpace(defaultActions))
+                {
+                    actions = client.ChooseRanking(eventId, context);
+                }
+                else
+                {
+                    int[] defaultActionArray = Array.ConvertAll(defaultActions.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries), Convert.ToInt32);

```

Comment:
I quickly checked var x = ",".Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries) which yield string[0] and int[0] subsequently. 
Is ChooseRanking() fine with int[0]?